### PR TITLE
Add metabase integration.

### LINKED ISSUE #1682: document metabase support
### Related code PR

https://github.com/risingwavelabs/risingwave/issues/2613 

### Which part(s) of the docs might be affected or should be updated? And how?

The draft: https://www.notion.so/risingwave-labs/Connect-Metabase-to-RisingWave-Draft-69ad5956719a4e8eac199bedfd640bf4?pvs=4

### Reference

_No response_

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ Change integration Metabase's availability to "available".]
  - [Add Connect Metabase to RisingWave.md]
  - [Need to add an image of successful connection.]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ https://github.com/risingwavelabs/risingwave/issues/2613 ]

- **Related doc issue**
  
  Resolves [ https://github.com/risingwavelabs/risingwave-docs/issues/1682#event-11336863198]

## For reviewers

- **Preview**

  - [ https://pr-1688.d2fbku9n2b6wde.amplifyapp.com/docs/dev/metabase-integration/]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/guides/metabase-integration.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+<img
+src={require('../images/connection-details.png').default}
+alt="Fill in the Connection Details"
+/>
+    
+4. Save the connection.
+
+Once connected, you will see the RisingWave database available in Metabase. You can now build dashboards, charts, and graphs on top of the real-time data in RisingWave.
+
```
Comment (neverchanje) [inline docs/guides/metabase-integration.md]: I just updated this part in the notion draft and resolved Heng's comments. Please take a look.

**Review by neverchanje — APPROVED** [review]
Rest LGTM.

File: docs/guides/metabase-integration.md (hunk 1 above)
Comment (ShanlanLi) [reply docs/guides/metabase-integration.md]: Thanks, I've added the new part.

File: docs/guides/metabase-integration.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<img
+src={require('../images/add-a-database.png').default}
+alt="Add a Database"
+/>
+
+2. For the database type, select "PostgreSQL" since RisingWave uses the PostgreSQL wire protocol.
+
+3. Fill in the connection details:
+    - Name: Choose a friendly name.
+    - Host: The hostname or IP address of the RisingWave database. The default **Host** is `localhost`.
```
Comment (neverchanje) [inline docs/guides/metabase-integration.md]: Please remove the defaults. These redundant defaults were originally generated by AI : 😊 which do not exist actually

**Review by WanYixian — APPROVED** [review]
LGTM

**Pull request merged**